README.md

```diff
@@ -1,7 +1,7 @@
-# TaskCat CI/CD pipeline for AWS CloudFormation Quick Start
+# TaskCat CI/CD pipeline for AWS CloudFormation Partner Solution
```

Comment:
Do we want to call this project a "Partner Solution" since there's no partner involved? If not, what will we call it. ?